### ISSUE TITLE
Implement Edit Applicant Info Screen

### ISSUE THREAD

**Issue by craigyu** [bcgov/nr-spar#596]
Dependent on #500 
Subtask of https://github.com/bcgov/nr-spar/issues/463

Requirements:

- [Figma interface](https://www.figma.com/file/xOTQWn4xwyIYGJQPo1OXvm/Registration-process-A-class---New-Version?type=design&node-id=2044-282817&mode=design&t=WRUwsDcylQCGXvSR-4) 

- Users should NOT be able to edit the applicant agency AND  location code
- Users should NOT be able to edit the seedlot species
- Users should be able to edit: The applicant email
- Users should be able to edit: A-class source
- Users should be able to edit: To be registered at the Tree Seed Centre
- Users should be able to edit: Collected from a location within B.C.
- The call to action button will allow users to Save edits

![Applicant   Lot information](https://github.com/bcgov/nr-spar/assets/99898407/44fddbf6-34d5-4700-9d21-fdffda9cda5d)

FYI, this feature will be used in B class seedlots, veglots

**Comment by craigyu** [comment]
duplicate of #501
